telos-frontend/src/components/todo/DashboardTodo.jsx

```diff
@@ -187,47 +298,99 @@ const DashboardTodo = () => {
         })}
       </List>
       <div>
-        <FormControl variant="outlined">
+        <FormControl className="inputbox" variant="outlined">
           <InputLabel htmlFor="outlined-adornment-password">New To Do</InputLabel>
           <OutlinedInput
             id="outlined-todo"
-            value={item.name}
+            value=""
             onChange={firstEvent}
             endAdornment={
               <InputAdornment position="end">
-                <IconButton className="AddBtn" onClick={item.name ? secondEvent : inputAlert}>
+                <IconButton className="AddBtn" onClick={openAdd}>
                   <AddIcon className="Publish" />
                 </IconButton>
               </InputAdornment>
             }
             labelWidth={70}
           />
         </FormControl>
-        <form className={classes.datecontainer} noValidate>
-          <TextField
-            id="date"
-            label="Due date"
-            type="date"
-            value={dueDate}
-            onChange={dateChange}
-            className={classes.datetextField}
-            InputLabelProps={{
-              shrink: true,
-            }}
-          />
-        </form>
       </div>
       <Menu
         id="simple-menu"
         anchorEl={anchorEl}
         keepMounted
         open={Boolean(anchorEl)}
         onClose={handleOptionClose}
+        style={{
+          alignItems: 'center',
+          textAlign: 'center',
+        }}
       >
-        <MenuItem onClick={handleOptionClose}>Option</MenuItem>
-        <MenuItem onClick={handleOptionClose}>Option</MenuItem>
-        <MenuItem onClick={handleOptionClose}>Option</MenuItem>
+        <MenuItem onClick={cancelEvent}>Cancel/Uncancel</MenuItem>
+        <MenuItem onClick={deleteEvent}>Delete</MenuItem>
+        <MenuItem onClick={openMigrate}>Migrate</MenuItem>
       </Menu>
+      <Dialog open={add} onClose={closeAdd} aria-labelledby="form-dialog-title">
+        <DialogTitle id="form-dialog-title">New To Do</DialogTitle>
+        <DialogContent>
+          <form className={classes.datecontainer} noValidate>
```

Comment:
are we deciding not to validate inputs? Is date impacted by this?

Reply:
there is a date picker there so not really ;)

Reply:
Also a little more commenting in the code to explain what certain things are would be useful